examples/interactable_cube.rs

```diff
@@ -78,19 +78,16 @@ fn interactable_demo(mut imesh_entities: Query<&InteractableMesh>) {
             println!("Mouse Exited");
         }
 
-        match imesh.mouse_down(MouseButton::Left) {
-            Some(v) => println!("Left Mouse Button is Down"),
-            None => (),
+        if imesh.mouse_down(MouseButton::Left).is_some() {
+            println!("Left Mouse Button is Down");
         }
 
-        match imesh.mouse_just_pressed(MouseButton::Left) {
-            Some(v) => println!("Left Mouse just Clicked"),
-            None => (),
+        if imesh.mouse_just_pressed(MouseButton::Left).is_some() {
+            println!("Left Mouse just Clicked");
         }
 
-        match imesh.mouse_just_released(MouseButton::Left) {
-            Some(v) => println!("Left Mouse just Released"),
-            None => (),
+        if imesh.mouse_just_released(MouseButton::Left).is_some() {
+            println!("Left Mouse just Released");
```

Comment:
Clippy was complaining about a single case `match` as well as `v` being unused and recommended that it was switched to if statement, let me know if this is ok, I can always revert it